vividus-tests/src/main/resources/story/integration/SetVariableSteps.story

```diff
@@ -42,6 +42,12 @@ When I save `name` attribute value of element located `By.id(text)` to SCENARIO
 Then `text` is equal to `${variableName}`
 
 
+Scenario: Verify step: "When I save `$attributeName` attribute value of element located `$locator` to $scopes variable `$variableName`" for element with shadowCssSelector
+When I perform javascript 'const shadow1 = document.querySelector(`#output`).attachShadow({mode: `open`}); shadow1.innerHTML = `<div></div>`; const shadow2 = shadow1.querySelector(`div`).attachShadow({mode: `open`}); shadow2.innerHTML = `<div class='shadow-element'></div>`; return 0;' and save result to the 'STEP' variable 'addShadow'
```

Comment:
I propose to modify the test site, there is no reason to modify the dom during the test because for example, it could mislead investigation if the javascript step is affected by reg regression